csi.proto

```diff
@@ -99,6 +93,58 @@ message GetPluginInfoResponse {
 }
 ////////
 ////////
+message GetPluginCapabilitiesRequest {
+  // The API version assumed by the CO. This is a REQUIRED field.
+  Version version = 1;
+}
+
+message GetPluginCapabilitiesResponse {
+  // All the capabilities that the controller service supports. This
+  // field is OPTIONAL.
+  // If any of the `CONTROLLER_xxx` RPC capabilities are specified then
+  // it is REQUIRED for a plugin to also report a `CONTROLLER_SERVICE`
+  // service capability as well.
+  repeated PluginCapability capabilities = 2;
+}
+
+// Specifies a capability of the plugin.
+message PluginCapability {
+  message RPC {
+    enum Type {
+      UNKNOWN = 0;
+      CONTROLLER_CREATE_DELETE_VOLUME = 1;
```

Comment:
I was under the impression that the `PluginCapability` was **only** going to enumerate whether or not the `Controller` or `Node` service was available and that the existing capability RPCs for the eponymous services were going to continue to be used for now. That's what I suggested and @saad-ali agreed in deference to keeping things simple and reducing the required changes at this time. I do not think it's a good idea, nor a necessary one, to pollute a single RPC with capabilities from both Node and Controller services.